### ISSUE TITLE
Note regarding old vs new Mixin syntax

### ISSUE THREAD

**Issue by dstorey** [heycam/webidl#536]
We're looking to move to the new mixin syntax in SVG. Unfortunately we link to spec(s) that haven't updated to the new syntax yet, which is somewhat blocking us from moving forward (we would include a mixin rather than implement, but it isn't a mixin in the spec where the mixin is defined).

@tobie mentions adding a note to explain that mismatch. Would it be possible to update the WebIDL spec to make it clear the new and old syntax are equivalent and thus it is ok for a newer spec to use the new syntax for including a mixin without the spec iti s defined in updating to the new syntax?

That would allow us to move forward faster (and presumably other spec that reference specs that are not updated often)

**Comment by dstorey** [comment]
This relates to https://github.com/w3c/svgwg/pull/376

**Comment by domenic** [comment]
I don't think such a note would be accurate. Although they are superficially similar, the semantics are very different, in terms of what an interface can do vs. what a mixin can do. A lot of capabilities were removed and simplified in making mixins.

**Comment by AmeliaBR** [comment]
What about a more generic note to the `includes` command, that as a transitionary measure, you can also include a `NoInterfaceObject` interface?

So, not saying that the two interface types are equivalent, but saying that they can both be included in other interfaces in an equivalent way.

**Comment by domenic** [comment]
That wouldn't be accurate either, though.

In the end, this is just software engineering: if your dependencies haven't updated themselves, you'll have a hard time updating yourself.

**Comment by annevk** [comment]
Yeah, I think you should just update and maybe add a note about out-of-date dependencies. Which dependencies though? Did you file issues with them?
